Resources/Views/Authentication/create_events.leaf

```diff
@@ -0,0 +1,48 @@
+#extend("base"):
+    #export("title", "SwiftLeeds")
+    #export("content"):
+        #extend("Shared/_header")
+        <div class="row">
+            <form method="POST" action="/api/v1/events/create" enctype="application/x-www-form-urlencoded">
+                <h1>Create presentation</h1>
+                <br>
+                <div class="mb-3">
+                  <label for="title" class="form-label">Event Name</label>
```

Comment:
Do we need Event in front of these as we know we're on a Create Event page, i.e. just Name?  Same for other labels